class_configs/brd_class_config.lua

```diff
@@ -740,73 +752,304 @@ local _ClassConfig = {
+                cond = function(self, songSpell)
+                    local setting = RGMercUtils.GetSetting('DoAEMez')
+                    local memmed = RGMercUtils.SongMemed(songSpell)
+                    local spellReady = mq.TLO.Me.SpellReady(songSpell)()
+                    local aeMezCt = RGMercUtils.GetSetting("AEMezCount")
+
+                    local res = setting and memmed and spellReady
+                        and RGMercUtils.GetXTHaterCount() >= aeMezCt
+
+                    return res
+                end,
+            },
+            {
+                name = "AESlowSong",
+                type = "Song",
+                cond = function(self, songSpell)
+                    return RGMercUtils.GetSetting("DoAESlow")
+                        and RGMercUtils.DetSpellCheck(songSpell)
+                        and RGMercUtils.GetXTHaterCount() > 2
+                        and not mq.TLO.Target.Slowed()
+                end,
+            },
+            {
+                name = "SlowSong",
+                type = "Song",
+                cond = function(self, songSpell)
+                    return RGMercUtils.GetSetting("DoSlow")
+                        and RGMercUtils.DetSpellCheck(songSpell)
+                        and not mq.TLO.Target.Slowed()
+                end,
+            },
+            {
+                name = "DispelSong",
+                type = "Song",
+                cond = function(self, songSpell)
+                    return RGMercUtils.GetSetting('DoDispel') and mq.TLO.Target.Beneficial()
+                end,
+            },
+            {
+                name = "Dreadstone",
+                type = "Item",
+                cond = function(self, itemName)
+                    -- This item is instant cast for free with almost no CD, just mash it forever when it's available
+                    return mq.TLO.FindItem(itemName).TimerReady() == 0
+                end,
+            },
         },
         ['Heal'] = {
         },
-        ['DPS'] = {
+
+        ['Testing'] = {
+            {
+                name = "FireDotSong",
+                type = "Song",
+                cond = function(self, songSpell)
+                    return RGMercUtils.SongMemed(songSpell) and RGMercUtils.GetSetting('UseFireDots') and
+                        -- If dot is about to wear off, recast
+                        getDetSongDuration(songSpell) <= 4
+                end,
+            },
+            {
+                name = "IceDotSong",
+                type = "Song",
+                cond = function(self, songSpell)
+                    return RGMercUtils.SongMemed(songSpell) and RGMercUtils.GetSetting('UseIceDots') and
+                        -- If dot is about to wear off, recast
+                        getDetSongDuration(songSpell) <= 4
+                end,
+            },
+        
         },
-        ['Downtime'] = {
+        ['DPSProcedural'] = {
+            doFullRotation = true,
```

Comment:
Procedural is the pre-existing bard config that was already there when I started, which I meant to add back into a different rotation people could choose instead of the dynamic one, then I code folded it away and forgot about it forever after.  Excellent catch